<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderPropertyConfigWidget</class>
 <widget class="QDockWidget" name="RenderPropertyConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Render Property</string>
  </property>
  <widget class="QWidget" name="content">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QLabel" name="relatedDataObject">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>(No object selected)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="timeSelectionWidget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="startDateLabel">
         <property name="text">
          <string>&amp;Start Date</string>
         </property>
         <property name="buddy">
          <cstring>startDateSlider</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSlider" name="startDateSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="startDateCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="endDateLabel">
         <property name="text">
          <string>&amp;End Date</string>
         </property>
         <property name="buddy">
          <cstring>endDateSlider</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="endDateSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="endDateCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="propertyguizeug::PropertyBrowser" name="propertyBrowser"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>propertyguizeug::PropertyBrowser</class>
   <extends>QTreeView</extends>
   <header>propertyguizeug/PropertyBrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
